.github/workflows/pr-gpu.yaml

```diff
@@ -18,6 +18,11 @@ jobs:
           markers: not daily and not remote and gpu and (doctest or not doctest)
           pytest_command: coverage run -m pytest
           composer_package_name: mosaicml
+        - name: gpu-3.11-2.2
+          container: mosaicml/pytorch:2.2.0_cu121-python3.11-ubuntu20.04
+          markers: not daily and not remote and gpu and (doctest or not doctest)
+          pytest_command: coverage run -m pytest
+          composer_package_name: mosaicml
```

Comment:
Let's keep only on daily for now to minimize load and only keep `-latest` tag (2.1) for per run?